setup.py

```diff
@@ -122,6 +122,7 @@ def build_extension(self, ext):
             "pytest-cov",
             "hilbertcurve==1.0.5",
             "hypothesis[numpy]==6.54.6",
+            "py3Dmol",
```

Comment:
pin the version?

Reply:
Good call, done in https://github.com/proteneer/timemachine/pull/1028/commits/07b26d68578938cb17b5e4c9a816fd323195d387